pgmoon/socket.moon

```diff
@@ -20,10 +20,8 @@ luasocket = do
       sock = @sock
       original = sock[key]
       if type(original) == "function"
-        fn = (_, ...) ->
+        (_, ...) ->
```

Comment:
Yes, this change has been worrying me too. I'll try to come up with a workaround.